src/jobflow_remote/jobs/jobcontroller.py

```diff
@@ -3189,6 +3190,7 @@ def update_flow_state(
         updated_states
             A dictionary with the updated states of Jobs that have not been
             stored in the DB yet. In the form {job_uuid: JobState value}.
+            If the value is None the Job is considered deleted.
```

Comment:
The Job is never really in the None state. The point here is just to signal that the state of the Job should be ignored, since it will be deleted in the update. I will try to clarify the usage.